src/views/Error.vue

```diff
@@ -0,0 +1,104 @@
+<template>
+  <div class="container">
+    <!-- 404 error starts -->
+    <div v-if="isPageNotFound">
+      <h1>404</h1>
+      <p>You have visited a wrong link</p>
+      <p>आप गलत लिंक पर आ गए हैं</p>
+    </div>
+    <!-- 404 error ends -->
+
+    <!-- Browser error starts -->
```

Comment:
Ah, same error page is handling 404 (not found) and notSupported? I am not fully on board -- why not use two components --, but chalega. If we keep adding errors and this keeps growing, we can always split it later